assets/styles.css

```diff
@@ -0,0 +1,814 @@
+  background-color: grey;
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+  margin-right: 15px;
+}
+
+.fas {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.fa-clock,
+.fa-share-alt,
+.fa-rss,
+.fa-envelope {
+  color: white;
+}
+
+.fab {
+  color: white;
+}
+
+.footer-list ul {
+  display: flex;
+  justify-content: center;
+  color: white;
+  margin-left: 0;
+  padding-left: 0;
+  width: 100%;
+  font-size: 0.7em;
+}
+
+.omited-div {
+  display: flex;
+}
+
+.omitted-p {
+  text-decoration: line-through;
+  margin-right: 10px;
+  color: #aaa;
+  font-size: 12px;
+  margin-bottom: 0;
+  margin-top: 17px;
+}
+
+.hamburger {
+  display: none;
+  position: absolute;
+  right: 10px;
+  top: 5px;
+  color: red;
+}
+
+.search-button {
+  position: absolute;
+  top: 10px;
+  right: 70px;
+  display: none;
+}
+
+.last-grid-img h2 {
+  color: black;
+  font-size: 1em;
+}
+
+@media screen and (max-width: 767px) {
+  nav {
+    background-image: url(./logo.png);
+    background-size: 20% 70px;
+    background-position: left;
+    background-repeat: no-repeat;
+    background-position-y: top;
+  }
+
+  nav .flex-ul-container-two,
+  nav .flex-ul-container,
+  nav > :nth-child(1) {
+    visibility: hidden;
+  }
+
+  nav :nth-child(3) {
+    height: 35px;
+  }
+
+  nav :nth-child(2) {
+    position: absolute;
+    display: none;
+  }
+
+  main > :nth-child(1) > :nth-child(1) {
+    height: 100%;
+  }
+
+  .hamburger {
+    display: block;
+  }
+
+  .grid-img,
+  .last-grid-img {
+    display: flex;
+    justify-content: flex-start;
+    margin-bottom: 10px;
+  }
+
+  .grid-img img,
+  .last-grid-img img {
+    width: 75px;
+    height: 80px;
+  }
+
+  .first-section {
+    margin-top: 50px;
+    display: block;
+  }
+
+  .first-section > article {
+    min-width: 300px;
+  }
+
+  .flex-div2 {
+    width: 100%;
+    max-width: 100%;
+    margin: 0;
+    display: block;
+  }
+
+  .small-background-img {
+    width: 100%;
+  }
+
+  .second-headings {
+    display: block;
+  }
+
+  .fas {
+    padding-top: 0;
+  }
+
+  .second-flex-div {
+    margin-bottom: 10px;
+  }
+
+  .second-flex-div i {
+    margin-top: 0;
+  }
+
+  .tnw-down i {
+    padding-top: 0;
+  }
+
+  .omitted-p {
+    margin-top: 0;
+  }
+
+  h3 {
+    margin-block-start: 0;
+    margin-block-end: 0;
+    margin-left: 5px;
+    font-size: 1em;
+  }
+
+  p {
+    margin-block-start: 0;
+    margin-block-end: 0;
+    margin-left: 5px;
+  }
+
+  .grid-img {
+    margin-bottom: 10px;
+  }
+
+  .non-flex-section .spiff-background,
+  .non-flex-section .venture-background {
+    margin-right: 0;
+  }
+
+  .last-grid-img h2 {
+    font-size: 0.9em;
+    margin-right: 10px;
+    margin-left: 10px;
+  }
+
+  .tnw-explanations h1 {
+    font-size: 1em;
+  }
+
+  .tnw-img {
+    display: none;
+  }
+
+  .second-headings h1 {
+    margin-bottom: 10px;
+    font-size: 1.1em;
+  }
+
+  .flex-div * {
+    font-size: 1em;
+  }
+
+  .search-button {
+    display: block;
+  }
```

Comment:
Same style definition of the selector on line 642. Group selectors, or create a new class.